components/org.apache.stratos.manager/src/main/database/Stratos-dbScript

```diff
@@ -0,0 +1,91 @@
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+CREATE SCHEMA IF NOT EXISTS `StratosManager` DEFAULT CHARACTER SET utf8 COLLATE utf8_general_ci ;
+USE `StratosManager` ;
+
+-- -----------------------------------------------------
+-- Table `StratosManager`.`Cluster`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `StratosManager`.`Cluster` (
```

Comment:
Table names and column names might need to use the same naming convention we currently use: upper case letters with underscore separated words. Please refer existing database files (stratos-distribution/dbscripts/)
Thanks